packages/syft/src/syft/core/tensor/autodp/gamma_tensor.py

```diff
@@ -1363,8 +1428,12 @@ def _sub(state: dict) -> jax.numpy.DeviceArray:
             max_max = self.max_vals.data - other.max_vals.data
             _min_val = np.minimum.reduce([min_min, min_max, max_min, max_max])
             _max_val = np.maximum.reduce([min_min, min_max, max_min, max_max])
-            min_val = lazyrepeatarray(data=_min_val, shape=self.shape)
-            max_val = lazyrepeatarray(data=_max_val, shape=self.shape)
+            min_val = lazyrepeatarray(
+                data=_min_val, shape=self.shape, data_type="min_val"
```

Comment:
Q: Interesting ,Any semantic reason behind the data_type attribute?

Reply:
The idea was that if `lazyrepeatarray.data` ever becomes a multidimensional array instead of a single value, we could check if it is a min_val or max_val lazyrepeatarray, and collapse the array down to a single value accordingly by calling max or min